src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -24,7 +25,7 @@ namespace Microsoft.CodeAnalysis.CodeFixes
     /// </summary>
     internal partial class BatchFixAllProvider : FixAllProvider
     {
-        public static readonly FixAllProvider Instance = new BatchFixAllProvider();
+        public static readonly BatchFixAllProvider Instance = new BatchFixAllProvider();
```

Comment:
undo?  shouldn't be necessary now that my PR is in right?

Reply:
Yes, I can undo this.